be/src/Project/Web/Seller/Seller.Web/Areas/Products/ApiRequestModels/SaveCategoryApiRequestModel.cs

```diff
@@ -7,8 +8,7 @@ namespace Seller.Web.Areas.Products.ApiRequestModels
     public class SaveCategoryApiRequestModel : RequestModelBase
     {
         public string Name { get; set; }
-        public string Schema { get; set; }
-        public string UiSchema { get; set; }
+        public IEnumerable<CategorySchema> Schemas { get; set; }
```

Comment:
**suggestion:** It still hasn't been changed. Please change to CategorySchemaRequestModel and after that. Please remove redundand using